ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/cmake/tasks/main.yml

```diff
@@ -40,6 +40,21 @@
     - (ansible_architecture == "x86_64") or (ansible_architecture == "ppc64le") or (ansible_architecture == "s390x")
   tags: cmake
 
+# ansible_processor_vcpus is not defined on Linux s390x
+# set it to {{ ansible_processor_cores }} if defined or to 1
+- name: Set ansible_processor_vcpus variable
+  set_fact:
+    ansible_processor_vcpus: "{{ ansible_processor_cores | default(1) }}"
+  when:
+    - ansible_processor_vcpus is not defined
+    - (cmake_installed.rc != 0 ) or (cmake_installed.rc == 0 and cmake_version.stdout is version_compare('3.11.4', operator='lt') )
+    - (ansible_architecture == "x86_64") or (ansible_architecture == "ppc64le") or (ansible_architecture == "s390x")
+
```

Comment:
If you're running the `cmake` role only, then there should be a `cmake` tag for this task no?

Reply:
I think the right fix might be to add the tag, and any others that might rely on it, to the task in ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/main.yml.

Reply:
Within the Unix playbook, the following are tags of the roles, excluding Common, that use ansible_processor_vcpus:

- `git_source`
- `ccache`
- `cmake`
- `gcc-4.8`
- `local_srcinstall` (though this role is to be removed via https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/1102)
- `nagios_plugins`

Reply:
Within the Unix playbook, the following are tags of the roles, excluding Common, that use ansible_processor_vcpus:

- `git_source`
- `ccache`
- `cmake`
- `gcc-4.8`
- `local_srcinstall` (though this role is to be removed via https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/1102)
- `nagios_plugins`